tests/kola/misc-ign-ro/test.sh

```diff
@@ -36,3 +36,27 @@ if [ "$context" != "system_u:object_r:net_conf_t:s0" ]; then
     fatal "SELinux context on stub-resolv.conf is wrong"
 fi
 ok "SELinux context on stub-resolv.conf is correct"
+
+# This is for verifying that `kubernetes_file_t` labeled files can be
+# watched by systemd
+# See: https://github.com/coreos/fedora-coreos-tracker/issues/861
+# See: https://github.com/containers/container-selinux/issues/135
+if [ "$(systemctl is-active kube-watch.path)" != "active" ]; then
+    fatal "kube-watch.path did not activate successfully"
+fi
+ok "kube-watch.path successfully activated"
+
+if [ "$(touch /etc/kubernetes/kubeconfig)" ]; then
+    fatal "Unable to create /etc/kubernetes/kubeconfig"
+fi
+ok "successfully created /etc/kubernetes/kubeconfig"
```

Comment:
this bit is confusing to me. I'm not sure we need an `if` statement at
all because we have `set -eux`.

```suggestion
touch /etc/kubernetes/kubeconfig
ok "successfully created /etc/kubernetes/kubeconfig"
```